cvat/apps/engine/task.py

```diff
@@ -60,29 +57,6 @@
 
 slogger = ServerLogManager(__name__)
 
-############################# Low Level server API
-
-def create(
-    db_task: models.Task,
-    data: models.Data,
-    request: ExtendedRequest,
-) -> str:
-    """Schedule a background job to create a task and return that job's identifier"""
-    q = django_rq.get_queue(settings.CVAT_QUEUES.IMPORT_DATA.value)
-    user_id = request.user.id
-    rq_id = RQId(RequestAction.CREATE, RequestTarget.TASK, db_task.pk).render()
-
-    with get_rq_lock_by_user(q, user_id):
-        q.enqueue_call(
-            func=_create_thread,
-            args=(db_task.pk, data),
-            job_id=rq_id,
-            meta=ImportRQMeta.build_for(request=request, db_obj=db_task),
-            depends_on=define_dependent_job(q, user_id),
-            failure_ttl=settings.IMPORT_CACHE_FAILED_TTL.total_seconds(),
-        )
-
-    return rq_id
 
 ############################# Internal implementation for server API
```

Comment:
I guess this comment can be deleted too, since it doesn't make sense without the first one.